Dockerfile

```diff
@@ -0,0 +1,27 @@
+ARG NODE_VERSION=14.x
+
+FROM gcr.io/connectedcars-staging/node-builder.master:$NODE_VERSION as builder
+
+ARG SSH_KEY_PASSWORD
+ARG NPM_TOKEN
```

Comment:
I don't think you need either of those, since this is not using any private packages.